pkg/api/handlers/compat/containers.go

```diff
@@ -403,6 +403,24 @@ func LibpodToContainerJSON(l *libpod.Container, sz bool) (*types.ContainerJSON,
 		state.Status = define.ContainerStateCreated.String()
 	}
 
+	state.Health = &types.Health{
+		Status:        inspect.State.Healthcheck.Status,
+		FailingStreak: inspect.State.Healthcheck.FailingStreak,
+	}
+
+	log := inspect.State.Healthcheck.Log
+
+	for _, item := range log {
+		var res *types.HealthcheckResult = &types.HealthcheckResult{}
```

Comment:
Can this be stated as

res:=&types.HealthcheckResult{}

Reply:
yep, pushing that change now.